src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -633,6 +632,11 @@ def __init__(self, symbolized=True):
     self.raw_frames = []
     self.last_frame_id = -1
 
+    if fuzz_target:
+      self.fuzz_target = fuzz_target
+    else:
+      self.fuzz_target = environment.get_value('FUZZ_TARGET')
```

Comment:
environment.get_value('FUZZ_TARGET') or fuzz_target ?

Reply:
Done! (but changed the order since fuzz_target argument should take precedence).